counterparty-core/counterpartycore/lib/sentry.py

```diff
@@ -29,6 +31,17 @@ def before_send(event, _hint):
     return event
 
 
+def filter_transactions(event, _hint):
```

Comment:
Do we need to filter client-side? Sentry says it'll do it server-side.

Reply:
yes we could just rename the transaction with a filtered name but for bandwith usage perhaps better to filter here ?

Reply:
I renamed the routes to "healthcheck", so should be filtered server side, and removed the filter client side.

Reply:
eh I think it's a tiny amount of bandwidth, and it's only for servers we host ourselves, really. Also if we don't filter client-side, we can turn the filter off server-side later if we want.